### PR TITLE
OCPNODE-1687: Adding conditional risk KubeletRestartBug for OCPBUGS-14605

### PR DESCRIPTION
**Pull request by LalatenduMohanty**

### CODE REVIEW

File: blocked-edges/4.12.20-KubeletRestartBug.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+to: 4.12.20
+from: .*
+url: https://issues.redhat.com/browse/OCPNODE-1687
+name: KubeletRestartBug
+message: |-
+  Kubelet restart will restart all associated containers,pods and other resources (e.g. virtual machines) associated with it.
```
Comment (petr-muller) [inline blocked-edges/4.12.20-KubeletRestartBug.yaml:6]:
```suggestion
  Kubelet restart will restart all containers, pods and other resources (e.g. virtual machines) associated with it.
```

**Review by petr-muller — COMMENTED** [review]
Based on the bug description I'd probably say something like this in the message:

"Pods are will be terminated on Kubelet restart if they consume any device, which disrupts CNV where Pods are carrying Virtual Machines"

**Comment by fabiand** [comment]
I can not comment on the syntax, but from the content (query and messgae) it looks ok
To me it is crucial that we get this out quickly in order to avoid that too many customers move to it.

File: blocked-edges/4.12.20-KubeletRestartBug.yaml [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+- type: PromQL
```
[the remark this answers is not in the document]
Comment (LalatenduMohanty) [reply blocked-edges/4.12.20-KubeletRestartBug.yaml:8]: Thanks, fixed it.

File: blocked-edges/4.13.2-KubeletRestartBug.yaml [hunk 3]
```diff
@@ -0,0 +1,13 @@
+to: 4.13.2
+from: .*
+url: https://issues.redhat.com/browse/OCPNODE-1687
+name: KubeletRestartBug
+message: |-
+Kubelet restart will restart all containers, pods and other resources (e.g. virtual machines) associated with it.
```
Comment (wking) [inline blocked-edges/4.13.2-KubeletRestartBug.yaml:6]: This line needs a two-space indent.

**Review by petr-muller — APPROVED** [review]

**Pull request merged**